src/pages/user/recent.vue

```diff
@@ -0,0 +1,348 @@
+<script setup lang="tsx">
+	type UserList = AdminGetUserInfoResponseDto["result"];
+	const message = useMessage();
+
+	const isOpenUserInfoModal = ref(false);
+	const isPassUser = ref(false);
+	const isDeleteUser = ref(false);
+	const searchUserUid = ref<number | null>(null);
+	const currentSortKey = ref<string | null>("uid");
+	const currentSortOrder = ref<"ascend" | "descend" | false>("ascend");
+	const defaultUserInfoData = {
+		uid: -1,
+		avatar: "",
+		userBannerImage: "",
+		userNickname: "",
+		username: "",
+		gender: "",
+		signature: "",
+		label: [],
+	};
+	const userInfoData = reactive({ ...defaultUserInfoData });
+	const genderMap: Record<string, string> = {
+		male: "男",
+		female: "女",
+		unknown: "未知",
+	};
+
+	const columns = computed<DataTableColumns<NonNullable<UserList>[number]>>(() => [
+		{
+			title: "UID",
+			key: "uid",
+			sorter: "default",
+			sortOrder: currentSortKey.value === "uid" ? currentSortOrder.value : false,
+		},
+		{
+			title: "UUID",
+			key: "UUID",
+		},
+		{
+			title: "昵称",
+			key: "userNickname",
+			sorter: "default",
+			sortOrder: currentSortKey.value === "userNickname" ? currentSortOrder.value : false,
+		},
+		{
+			title: "用户名",
+			key: "username",
+		},
+		{
+			title: "邮箱",
+			key: "email",
+		},
+		{
+			title: "角色",
+			key: "roles",
+			render(row) {
+				if (!row.roles?.length) return h(NTag, { style: { marginRight: "6px" }, type: "info", bordered: false }, { default: () => "user" });
+				const roles = row.roles.map(roleKey => {
+					return h(NTag, { style: { marginRight: "6px" }, type: "info", bordered: false }, { default: () => roleKey });
+				});
+				return roles;
+			},
+		},
+		{
+			title: "注册时间",
+			key: "userCreateDateTime",
+			sorter: "default",
+			sortOrder: currentSortKey.value === "userCreateDateTime" ? currentSortOrder.value : false,
+			render(row) {
+				if (row.userCreateDateTime === undefined) return h(NText, { depth: 3 }, () => "未记录");
+				const result = formatDateTime(row.userCreateDateTime);
+				if (!result) return h(NText, { depth: 3 }, () => "未记录");
+				return h("div", { class: "time-wrapper" }, [h("div", result.formatted), h(NText, { depth: 3, class: "text-xs" }, () => `(${result.relative})`),
+				]);
+			} },
+		{
+			title: "操作",
+			key: "actions",
+			render: row => (
+				<NFlex>
+					<NButton strong secondary size="small" type="info" onClick={() => openUserInfoModal(row)}>{{ icon: () => <Icon name="description" /> }}</NButton>
+					<NPopconfirm onPositiveClick={() => clearUserInfo(row.uid)}>
+						{{
+							trigger: () =>
+								<NButton type="error" strong secondary size="small">{{ icon: () => <Icon name="delete" /> }}</NButton>,
+							default: () => "确认驳回该用户修改信息审核吗？",
+						}}
+					</NPopconfirm>
```

Comment:
```suggestion
					<NPopconfirm onPositiveClick={() => clearUserInfo(row.uid)}>
						{{
							trigger: <NButton type="error" strong secondary size="small">{{ icon: <Icon name="delete" /> }}</NButton>,
							default: "确认驳回该用户修改信息审核吗？",
						}}
					</NPopconfirm>
```

不确定改得对不对（能不能正常跑），如果不对请忽略。